package.json

```diff
@@ -194,6 +195,7 @@
     "typescript": "5.7.3",
     "typescript-eslint": "^8.26.1",
     "webpack": "5.98.0",
+    "yaml-eslint-parser": "^1.3.0",
```

Comment:
I think you should be able to remove this. `eslint-plugin-yml` is self contained